libs/langchain-google-common/src/tests/llms.test.ts

```diff
@@ -521,4 +521,30 @@ describe("Mock Google LLM", () => {
     expect(responseArray).toHaveLength(3);
     console.log("record", JSON.stringify(record, null, 2));
   });
+
+  test("8: streamGenerateContent - streaming - json responseMimeType", async () => {
```

Comment:
Can we also get a test to make sure the request contains the `responseMimeType` setting in the generation config?